app/styles/component_tree.scss

```diff
@@ -101,6 +101,18 @@
   .value-token {
     color: var(--spec03);
   }
+
+  .arg-value-token {
+    background: none;
+    border: none;
+    padding: 0;
+  }
+
+  .arg-value-token:hover {
+    background-color: var(--spec03);
+    color: var(--white);
```

Comment:
Instead of using a hardcoded color, try using one of the `--base` or `--spec` colors here (e.g. `--base00`), that way you will avoid the white-on-yellow low contrast problems when switching between light and dark themes. See https://github.com/emberjs/ember-inspector/blob/master/app/styles/colors.scss @nummi may have more concrete suggestion on which combination to pick.

Reply:
The white-on-yellow contrast issue is probably a blocker, but should be an easy fix, it may or may not be the best looking option but switching to `--base00` should at least resolve the contrast problem.